apps/integration/int-rucio-authserver.yaml

```diff
@@ -5,3 +5,5 @@ ingress:
 
 httpd_config:
   grid_site_enabled: "True"
+
+exposeErrorLogs: False
```

Comment:
@ericvaandering 
Ah, what is this setting for? Is this necessary for 33?
Did I turn it on? I can't recall